semgrep_output_v1.atd

```diff
@@ -917,6 +917,7 @@ type found_dependency = {
   ?line_number: int option;
   (* store child dependency information for calculating path to transitivity*)
   ?children: dependency_child list option;
+  ?ref: string option
```

Comment:
please add a comment before like (* since 1.66.0 *).

Reply:
also I have no idea what a ref means, so maybe an example would be useful.

Reply:
updated naming, and added doc